lib/iris/tests/unit/experimental/ugrid/test_CFUGridMeshVariable.py

```diff
@@ -31,6 +31,22 @@ class TestIdentify(tests.IrisTest):
     def setUp(self):
         self.cf_identity = "mesh"
 
+    def test_cf_role(self):
+        match_name = "match"
```

Comment:
Worth some comments, I think, explaining in one line what the different testcases (existing and new) are doing.
If I have it right, something like:
  * test_cf_identity : a variable is IDd as a mesh-variable when it appears in some `data-variable::mesh`, even if it does not have a `var::cf_role == 'mesh_topology'`
  * test_cf_role : a variable is IDd as a mesh-variable when it has `var::cf_role == 'mesh_topology'`, even if it does **_not_** appear in some `data-variable::mesh`
  * test_cf_role_and_identity : a variable is IDd as a mesh-variable when it has `var::cf_role == 'mesh_topology;`, **_and_** appears in some `data-variable::mesh`

Reply:
ff73cb0308cba274c1ab50af990cf2dd835c4833